sdk/test/unit/context.test.ts

```diff
@@ -284,19 +284,12 @@ describe('Context instances', () => {
     for (const provider of context.web3Providers) {
       expect(provider).toBeInstanceOf(JsonRpcProvider);
       expect(provider.connection.url).toBe('https://goerli.base.org/');
-      provider
-        .getNetwork()
-        .then(nw => {
-          expect(nw.chainId).toEqual(84531);
-          expect(nw.name).toEqual('baseGoerli');
-          expect(nw.ensAddress).toEqual(
-            LIVE_CONTRACTS[SupportedVersion.LATEST].baseGoerli
-              .ensRegistryAddress
-          );
-        })
-        .catch(err => {
-          throw err;
-        });
+      const network = provider.network;
+      expect(network.chainId).toEqual(84531);
+      expect(network.name).toEqual('baseGoerli');
+      expect(network.ensAddress).toEqual(
+        LIVE_CONTRACTS[SupportedVersion.LATEST].baseGoerli.ensRegistryAddress
```

Comment:
why is this changed in the PR?

Reply:
Used to be handled with a then, for some reason that failed so i deleted it, then the indent on this changed that why its showing up as changes

Reply:
Is this the run that failed? https://github.com/aragon/osx-commons/actions/runs/8095266336/job/22121537332#step:6:176

I encountered the same error [in another PR](https://github.com/aragon/osx-commons/actions/runs/8065527304/job/22031515424#step:6:176) and mentioned it in our [discord thread here](https://discord.com/channels/672466989217873929/1212050307530367027/1212050309115940966).
I would like to understand this more. Is this really fixed with this change?

Reply:
Yes, this fixes the issue, however this is part of the sdk 1.0. I plan to remove this from thee commons after I have all tested and sorted